cmd/fyne/commands/install.go

```diff
@@ -23,43 +79,10 @@ type installer struct {
 }
 
 // NewInstaller returns an install command that can install locally built Fyne apps.
-func NewInstaller() Command {
+func NewInstaller() *installer {
```

Comment:
Is this func called from other packages, does it need to be exported or even exist?